ocs_ci/ocs/ui/base_ui.py

```diff
@@ -417,6 +421,16 @@ def login_ui():
     wait = WebDriverWait(driver, 60)
     driver.maximize_window()
     driver.get(console_url)
+    if config.ENV_DATA["flexy_deployment"]:
```

Comment:
I believe this is generic enough so it should work with both flexy and non-flexy and hence the IF statement can be removed